config/config.txt

```diff
@@ -632,3 +632,7 @@ PR_ANNOUNCEMENTS_PER_ROUND 5
 
 ## Uncomment to block granting profiling privileges to users with R_DEBUG, for performance purposes
 #FORBID_ADMIN_PROFILING
+
+## Comment to disable sending a toast notification on the host server when initializations complete.
+## Even if this is enabled, a notification will only be sent if there are no clients connected.
+TOAST_NOTIFICATION_ON_INIT
```

Comment:
i think you need to comment this to make it disabled by default

Reply:
there are two defaults.

code defaults (for if the config doesn't exist in the config file, like on existing servers) and config file defaults, which apply to new copies of the game folder, like those in dev builds.\

They have the config correct for the intended function of being enabled on dev builds but disabled on any prod server that pulls down the changes.